Week-10/url-shortner/src/LinkResult.js

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react'
+import {CopyToClipboard} from 'react-copy-to-clipboard';
+import axios from 'axios';
+
+const LinkResult = ({inputValue}) => {
+   
+    const [shortenLink,setShortenLink] = useState('')
+    const [copy,setCopy] = useState(false)
+    const [loading,setLoading] = useState(false)
+    const [error,setError] = useState(false)
+
+    useEffect(()=>{
+        const timer = setTimeout(()=>{
+          setCopy(false)
```

Comment:
why is this required?

Reply:
To provide the feedback to the user upon click of the "SHORTEN" button